modules/my_health/spec/request/v1/prescriptions_request_spec.rb

```diff
@@ -130,6 +130,18 @@
         expect(JSON.parse(response.body)['meta']['pagination']['per_page']).to eq(10)
       end
 
+      it 'responds to GET #refill_list with list of refillable prescriptions' do
```

Comment:
I think you may have changed the method name away from `refill_list` at some point.

Reply:
@kpethtel I will fix this up, thanks for pointing it out.